app/views/studies/initialize_study.html.erb

```diff
@@ -1,15 +1,4 @@
-<div class="row">
```

Comment:
Can we get this whole block back in?  We lost the 'View Study' button, which was something that got added recently that we definitely want to keep.  This may have just been a merge issue.